### PR TITLE
Add Makefile Support

### PR DESCRIPTION
**Pull request by arpitbatra123**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1 @@
+nothing:
```
Comment (kdichev) [inline Makefile]: are we sure that using nothing: aligns with our code style?

Comment (rakshazi) [reply Makefile]: Hm..... I think, it should be at least uppercased, but the better way is just cleanup that file, IMHO.

Comment (SevastyanDark) [reply Makefile]: @rakshazi Agree

**Comment by zorts** [comment]
LGTM:

```
jcallen@myhost:~/nocode> make
make: Nothing to be done for 'nothing'.
jcallen@myhost:~/nocode> 
```

**Pull request closed without merging**